conf.py

```diff
@@ -108,9 +108,9 @@ def _asset_hash(path: os.PathLike[str]) -> str:
 ]
 
 rediraffe_redirects = {
-    "clang.rst": "advanced-tools/clang.rst",
-    "coverity.rst": "advanced-tools/coverity.rst",
-    "gdb.rst": "advanced-tools/gdb.rst",
+    "clang.rst": "development-tools/clang.rst",
+    "coverity.rst": "development-tools/coverity.rst",
+    "gdb.rst": "development-tools/gdb.rst",
```

Comment:
> @hugovk, should I also do something with the redirects dict? (Would it be possible to link from the old URL to the new URL?)

Yes, if we add something like this:

```suggestion
    "clang.rst": "development-tools/clang.rst",
    "advanced-tools/clang.rst": "development-tools/clang.rst",
    "coverity.rst": "development-tools/coverity.rst",
    "advanced-tools/coverity.rst": "development-tools/coverity.rst",
    "gdb.rst": "development-tools/gdb.rst",
    "advanced-tools/gdb.rst": "development-tools/gdb.rst",
```

So for example, `/advanced-tools/clang.html` will redirect to `/development-tools/clang.html`.

This is testable locally too.

Reply:
Neat; thanks!

Reply:
I added some comments: cb28456e6a023f10a8da2bd8335b06fbd305de5c Hope that's ok.